lib/file_transfer_agent/azure_util.js

```diff
@@ -26,7 +29,7 @@ function AzureLocation(containerName, path) {
  * @returns {Object}
  * @constructor
  */
-function AzureUtil(azure, filestream) {
+function AzureUtil(connectionConfig, azure, filestream) {
```

Comment:
is it our public api? it looks like we are adding required parameter here

Reply:
It's tricky about what is public. I can sign this change in the scope of BCR.